src/app/forms/components/base-control/base-control.component.ts

```diff
@@ -31,6 +33,9 @@ export class BaseControlComponent implements ControlValueAccessor, AfterContentI
   ngAfterContentInit(): void {
     const ngControl: NgControl | null = this.injector.get(NgControl, null);
     if (ngControl) {
+      if (!ngControl.control) {
```

Comment:
Was this just for local debugging, or is it useful in production?

Reply:
Local. Removed. Thanks 🙂